src/main.rs

```diff
@@ -12,8 +12,9 @@ struct Cli {
     user_nation: String,
 
     /// Name of the output file
-    #[arg(short, long, default_value = "srsglass.xlsx")]
-    outfile: String,
+    // #[arg(short, long, default_value = "srsglass.xlsx")]
+    #[arg(short, long)]
```

Comment:
```suggestion
    #[arg(short, long, default_value = format!("srsglass{}.xlsx", chrono::Local::now().date_naive()))]
```
I think this should work with the `string` feature of the `clap` crate enabled.